DirectProgramming/C++SYCL/SYCL-Graph/dot-product/src/dotProduct.cpp

```diff
@@ -0,0 +1,90 @@
+//==============================================================
+// Copyright © 2025 Codeplay Software
```

Comment:
```suggestion
// Copyright © 2022 - 2023 Intel Corporation
```
This example was first published as part of the initial SYCL Graph extension.
https://github.com/intel/llvm/blob/e1163ff6500ba548b756d999f6503d58ffffcc80/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#81-dot-product